### PR TITLE
Fixed sanitizer on pasting HTML

### PR DESCRIPTION
**Pull request by woren951**
When I paste HTML text, sanitizer remove all inline tags and render only inner text. Because to HTMLJanitor passing wrong config.

### CODE REVIEW

**Comment by alxndr** [comment]
Any ETA for this fix?

**Comment by pawelbajgrowicz** [comment]
@neSpecc @khaydarov @gohabereg are you planning to check this pull request in the near future? This would be really great :rocket:

**Comment by Broutard** [comment]
Any news on it ? The bug appeared since 2.20.0

**Comment by swapster** [comment]
@gohabereg @khaydarov @neSpecc, Hi! Could you please check this PR?

**Comment by Naroh091** [comment]
Also waiting for this to get merged 🙏

File: src/components/modules/paste.ts [hunk 1]
```diff
@@ -662,7 +662,7 @@ export default class Paste extends Module {
 
     /** If there is no pattern substitute - insert string as it is */
     if (BlockManager.currentBlock && BlockManager.currentBlock.currentInput) {
-      const currentToolSanitizeConfig = BlockManager.currentBlock.tool.sanitizeConfig;
+      const currentToolSanitizeConfig = BlockManager.currentBlock.tool.baseSanitizeConfig;
```
Comment (neSpecc) [inline src/components/modules/paste.ts:665]: looks like the `sanitizeConfig` works wrong and we would better to fix it. Need to dig deeper.

**Review by neSpecc — CHANGES_REQUESTED** [review]

**Comment by neSpecc** [comment]
Looks like the PR is correct, but we have a problem with naming. See #2029

**Review by gohabereg — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**